src/routers/transactions/products.ts

```diff
@@ -89,8 +121,24 @@ router.get('/:type', async (req, res) => {
   }
  */
 router.post('/:type', async (req, res) => {
-  res.status(StatusCode.CREATED).send('Successfully created new product <type> transaction');
-}, authorized(UserType.ADMIN));
+  const createReq: ProdTransactionReq = {
+    productType: req.params.type,
+    datetime: req.body.datetime,
+    toPersonUuid: req.body.toPersonUuid,
+    fromPersonUuid: req.body.fromPersonUuid,
+    amountOfProduct: req.body.amountOfProduct,
+    costPerUnit: req.body.costPerUnit,
+    currency: req.body.currency,
+  };
+
+  if (req.body.milkQuality) {
+    createReq.milkQuality = req.body.milkQuality;
+  }
+
+  const createResult = await ProdTransactionsService.createProdTransactionsInDb(createReq);
+  const uuid = createResult[0];
```

Comment:
how come createResult is an array? might be better to return the first element instead of the whole array in createProdTransactionsInDb so that it cannot be misused